src/management/src/management_frontend.hpp

```diff
@@ -458,7 +457,7 @@ class RequestHandler {
     parse_json(json, d);
 
     VersionedModelId model_id = std::make_pair(get_string(d, "model_name"),
```

Comment:
This is the right place to validate that model names and versions don't contain restricted characters.